go/pkg/tree/tree.go

```diff
@@ -14,11 +13,17 @@ import (
 	"github.com/bazelbuild/remote-apis-sdks/go/digest"
 	"github.com/bazelbuild/remote-apis-sdks/go/pkg/chunker"
 	"github.com/bazelbuild/remote-apis-sdks/go/pkg/command"
+	"github.com/bazelbuild/remote-apis-sdks/go/pkg/filemetadata"
 	"github.com/golang/protobuf/proto"
 
 	repb "github.com/bazelbuild/remote-apis/build/bazel/remote/execution/v2"
 )
 
+// FileMetadataCache is a cache for file contents->Metadata.
+type FileMetadataCache interface {
+	Get(string) (*filemetadata.Metadata, error)
```

Comment:
nit: "path string" so its clear this receives file path and not contents (which could be implied by the comment above).

Reply:
Done.